engine/src/main/java/org/pentaho/di/trans/steps/infobrightoutput/InfobrightLoaderData.java

```diff
@@ -22,23 +22,22 @@
 
 package org.pentaho.di.trans.steps.infobrightoutput;
 
-import java.io.FileOutputStream;
-import java.io.OutputStream;
-import java.nio.charset.Charset;
-import java.sql.Connection;
-import java.sql.SQLException;
-
+import com.infobright.etl.model.BrighthouseRecord;
```

Comment:
Nothing material changed in this class - please remove from PR

Reply:
Class reverted (imports moved back, whitespace added back so it looks like it did previously).